python/generate/templates/api.mustache

```diff
@@ -1,248 +0,0 @@
-# coding: utf-8
```

Comment:
Where has this template gone? Again we look to have lost some functionality that was in the template as per (https://github.com/finbourne/lusid-sdk-python-preview/pull/105/files#diff-d43e1e36d080061a63324229c002c18d17a8ae70fd81f2cecbbc33258fbd1b44L173-L175)